cli/src/template_builder.rs

```diff
@@ -268,6 +279,29 @@ impl<'a, L: TemplateLanguage<'a>> CoreTemplateBuildFnTable<'a, L> {
         }
     }
 
+    pub fn empty() -> Self {
+        CoreTemplateBuildFnTable {
+            string_methods: HashMap::new(),
+            boolean_methods: HashMap::new(),
+            integer_methods: HashMap::new(),
+            signature_methods: HashMap::new(),
+            timestamp_methods: HashMap::new(),
+            timestamp_range_methods: HashMap::new(),
+        }
+    }
+
+    pub fn merge(&mut self, extension: CoreTemplateBuildFnTable<'a, L>) {
+        merge_fn_map(&mut self.string_methods, extension.string_methods);
```

Comment:
nit: (at your option) `extension` argument can be destructured to prevent future bugs.
```rust
let CoreTemplateBuildFnTable { string_methods, .. } = extension;
```

Reply:
Did it for both